src/vorta/views/repo_add_dialog.py

```diff
@@ -141,7 +141,7 @@ def validate(self):
         if RepoModel.get_or_none(RepoModel.url == self.values['repo_url']) is not None:
             self._set_status(self.tr('This repo has already been added.'))
             return False
-                      
```

Comment:
I really think we should move to something like black at some point.
The manual work to make flake8 happy is annoying to everyone^^

Reply:
Can do. There is also autopep8, which is slightly less aggressive than Black. I use that mainly.

Reply:
Fine as well, it just needs to be aggressive enough to satisfy flake8.
flake8 should probably still be in our CI, since it not only finds style errors :)

Do you have experience with flake8+autopep8?
I know that black and flake8 play well together as long as you force flake8 to accept lines with length <88

Reply:
After autopep8, all code should pass flake8.

For the last new project I used autopep8 + pylint&flake8. The latter two kind of complement each other. But flake8 by itself is also fine.

Reply:
Perfect!

Reply:
Just looked up, how to integrate autopep8 into our CI, but I don't get it.
I would have suspected that autopep8 has something similar to ```black --check```, but I can't find it.
@m3nu if you know how to do it, please open the PR :)

Reply:
I think a developer would run autopep before committing his code. It doesn't run automatically, since the changes it makes need to be reviewed.

Reply:
Absolutely, but the CI should fail if autopep *would* have changed something.
Otherwise, the following PRs will include changes by autopep which have nothing to do with you

Reply:
True. That's probably possible. Would be surprised if not.

Reply:
Agree, if you find it, please tell me.
Otherwise, I'd like to open a PR for black :)

Reply:
This would return exit code 2, if changes are made, but not change anything.

`autopep8 --exit-code --diff tests/*`

But I'm not sure 3 tools are needed to report errors. And since autopep8 fixes errors reported by [pycodestyle](https://github.com/PyCQA/pycodestyle), that and flake8 may be enough to run automatically. And maybe pylint because it checks some extra stuff. But not sure how big the difference is.

Reply:
Does autopep8 currently change anything, when being run on Vorta?

Reply:
I would guess no, since the code passes flake8.

Reply:
Just checked, it indeed changes nothing :)